CHANGES.rst

```diff
@@ -97,6 +97,8 @@ astropy.wcs
   and the `WCSTools documentation <http://tdc-www.harvard.edu/software/wcstools/wcstools.multiwcs.html>`_.
   [#6990]
 
+- Add WCS.contains function to check if the WCS footprint contains a given sky coordinate. [#7273]
```

Comment:
Minor nitpick: If you render the function as 
```
``WCS.contains()``
```
it ends up looking a bit more like a function call when rendered in the html changelog.